### PR TITLE
feat(@vtmn/svelte): add `VtmnCard` component

### PR DESCRIPTION
**Pull request by Tlahey**
Add a VtmnCard component
- Add 19 component tests

### CODE REVIEW

**Comment by GaspardMathon** [comment]
You missed on variant for the full-image : The overlay can be a gradient or a single color opacity : 
image

**Comment by GaspardMathon** [comment]
Overall it seems to work very well :) Thanks a lot

File: packages/sources/svelte/src/components/structure/VtmnCard/VtmnCard.svelte [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    variant === VARIANTS.FULL_IMAGE &&
+      contentOpaque &&
+      'vtmn-card_content--opaque',
+  );
+</script>
+
+<div class={componentClass}>
+  {#if img}
+    <div class={componentImageClass}>
+      <img src={img} alt="" />
```
Comment (Tlahey) [inline packages/sources/svelte/src/components/structure/VtmnCard/VtmnCard.svelte:40]: Use a slot instead a props ? 🤔

**Comment by Tlahey** [comment]
Ready to review and merge 🚀  !

File: packages/showcases/svelte/stories/components/structure/VtmnCard/VtmnCard.stories.svelte [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+    },
+    title: {
+      type: { name: 'string', required: true },
+      description: 'Title of the card.',
+      defaultValue: 'Card title',
+      control: {
+        type: 'text',
+      },
+    },
+  };
```
Comment (GaspardMathon) [inline packages/showcases/svelte/stories/components/structure/VtmnCard/VtmnCard.stories.svelte:40]: You should try to put the args of the component inside the .csf file to share the args with all libraries.

**Review by GaspardMathon — CHANGES_REQUESTED** [review]

**Review by GaspardMathon — APPROVED** [review]

**Review by lauthieb — APPROVED** [review]
Thanks a lot!!

**Pull request merged**